contracts/manifold/burnredeem/IBurnRedeemCore.sol

```diff
@@ -90,6 +90,7 @@ interface IBurnRedeemCore is IERC165, IERC721Receiver, IERC1155Receiver  {
         uint32 redeemedCount;
         uint16 redeemAmount;
         uint32 totalSupply;
+        uint8 contractVersion;
```

Comment:
Does this need to be in the middle? dapps will often read structs as arrays, this will shift indices.

Reply:
ethers will decode it properly if given the abi.  Yes, needs to be in the middle for struct packing (efficiency).  So ya, we need a conditional.